bciers/package.json

```diff
@@ -17,7 +20,9 @@
     "reg1:e2e:ui": "nx run registration1:e2e:ui",
     "reg1:e2e:report": "nx run registration1:report",
     "reg1:coverage": "nx run registration1:test --coverage",
-    "reg1:test": "nx run registration1:test"
+    "reg1:test": "nx run registration1:test",
+    "report": "nx dev reporting  -p 5000",
```

Comment:
Hey just playing with this branch and I noticed that my port 5000 is taken up and I had problems killing it. Turns out 5000 and 7000 are taken on Macs due to this annoying feature I've turned off: https://developer.apple.com/forums/thread/682332

Might want to change this to 6000 or something to make it easier for other devs, if not I can refer them to this so they can turn off airplay :)